client/blocks/app-promo/index.jsx

```diff
@@ -1,55 +1,85 @@
 import { Card, Button } from '@automattic/components';
 import { useLocalizeUrl } from '@automattic/i18n-utils';
+import classNames from 'classnames';
 import { useTranslate, useRtl } from 'i18n-calypso';
 import wpToJpImageRtl from 'calypso/assets/images/jetpack/wp-to-jp-rtl.svg';
 import wpToJpImage from 'calypso/assets/images/jetpack/wp-to-jp.svg';
+import QrCode from 'calypso/blocks/app-promo/qr-code';
 import AppsBadge from 'calypso/blocks/get-apps/apps-badge';
 import CardHeading from 'calypso/components/card-heading';
 import userAgent from 'calypso/lib/user-agent';
-
+import { useSelector } from 'calypso/state';
+import { isUserLoggedIn } from 'calypso/state/current-user/selectors';
 import './style.scss';
 
-export const AppPromo = () => {
+export const AppPromo = ( {
+	title = false,
+	iconSize = 29,
+	source = 'calypso-customer-home',
+	subheader = false,
+	className = 'customer-home__card',
+	hasQRCode = false,
+} ) => {
 	const isRtl = useRtl();
 	const translate = useTranslate();
 	const localizeUrl = useLocalizeUrl();
-	const { isDesktop, isiPad, isiPod, isiPhone, isAndroid } = userAgent;
+	const isLoggedIn = useSelector( isUserLoggedIn );
+	const iconWidth = Math.ceil( ( 49 / 29 ) * iconSize );
+
+	const { isDesktop, isiPad, isiPod, isiPhone, isAndroid, isMobile } = userAgent;
 	const isIos = isiPad || isiPod || isiPhone;
 
-	const showIosBadge = ! isAndroid && ! isDesktop;
-	const showAndroidBadge = ! isIos && ! isDesktop;
-	const showBadge = showIosBadge || showAndroidBadge;
+	const showIosBadge = ! isAndroid && ! isDesktop && userAgent;
+	const showAndroidBadge = ! isIos && ! isDesktop && userAgent;
+	const showBadge = ( showIosBadge || showAndroidBadge ) && isMobile;
+	if ( title === false ) {
+		title = translate( 'Get our mobile app' );
+	}
+
+	if ( subheader === false ) {
+		subheader = translate( 'Everything you need to build and grow your site from any device.' );
+	}
+
+	className = classNames( className, 'app-promo' );
```

Comment:
I'd just inline this at `<Card className={ classNames( className, 'app-promo' ) }>`

Reply:
good call!